daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -67,11 +67,15 @@ authorizationHeaders = [("Authorization", "Bearer " <> T.encodeUtf8 hardcodedTok
 
 withDamlServiceIn :: FilePath -> String -> [String] -> IO a -> IO a
 withDamlServiceIn path command args act = withDevNull $ \devNull -> do
-    let proc' = (shell $ unwords $ ["daml", command] <> args)
-          { std_out = UseHandle devNull
-          , create_group = True
-          , cwd = Just path
-          }
+    let proc' =
```

Comment:
Given that the integration tests are more or less testing what we expect users to use, we should migrate those pre 2.0. I don’t know what scenarios we use here but hopefully just a matter of replacing a few scenarios by init scripts. Let’s do it in a separate PR.

Reply:
yes, this one has a `TODO`